### PR TITLE
Add new slider style

### PR DESCRIPTION
**Pull request by pochretien**

### CODE REVIEW

File: scss/controls/slider.scss [hunk 1]
```diff
@@ -0,0 +1,67 @@
+.vapor-slider.rc-slider {
```
Comment (MichelML) [inline scss/controls/slider.scss]: could you remove `.vapor-slider.rc-slider {` and put each rule inside it at the first level, aka making this flat.

File: scss/controls/slider.scss [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  .rc-slider-dot-active {
+    height: $dot-active-height;
+    margin-top: initial;
+  }
+
+  .rc-slider-handle {
+    width: $handle-width;
+    height: $handle-height;
+    margin-top: -6px;
+    margin-left: -18px;
```
Comment (MichelML) [inline scss/controls/slider.scss]: variable for these two

File: scss/controls/slider.scss [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    background: url('../../resources/icons/svg/thumb.svg');
+    background-repeat: no-repeat;
+    background-size: $handle-background-size;
+    box-shadow: $handle-box-shadow;
+    border: none;
+    border-radius: $handle-border-radius;
+  }
+}
+
+.vapor-slider-overlay {
```
Comment (MichelML) [inline scss/controls/slider.scss]: could you write this part like this

```scss
.vapor-slider-overlay {
  padding: 0;
}

.rc-slider-tooltip-arrow {
  display: none;
}

.rc-slider-tooltip-inner {
  color: $dark-blue;
  font-size: $small-font-size;
  line-height: $default-line-height;

  background-color: transparent;
  box-shadow: none;
}
```

**Comment by MichelML** [comment]
can we see a picture of the style ?

File: scss/controls/slider.scss (hunk 1 above)
Comment (MichelML) [inline scss/controls/slider.scss]: discussed with @pochretien , @pochretien i would consider using only `.vapor-slider` or `.rc-slider` here if you can . Also, I would consider adding a comment at the top of the file saying you are overriding the default style of the library.

File: resources/icons/svg/thumb.svg [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  <style type="text/css">
+    .st0{filter:url(#filter-2);}
+    .st1{fill:#FFFFFF;}
+    .st2{fill:none;stroke:#F0F0F0;stroke-width:0.8;stroke-linecap:square;}
+  </style>
+  <filter filterUnits="objectBoundingBox" height="142.9%" id="filter-2" width="121.4%" x="-10.7%" y="-21.4%">
+    <feOffset dx="0" dy="0" in="SourceAlpha" result="shadowOffsetOuter1"></feOffset>
+    <feGaussianBlur in="shadowOffsetOuter1" result="shadowBlurOuter1" stdDeviation="1"></feGaussianBlur>
+    <feColorMatrix in="shadowBlurOuter1" type="matrix" values="0 0 0 0 0   0 0 0 0 0   0 0 0 0 0  0 0 0 0.3 0"></feColorMatrix>
+  </filter>
```
Comment (GermainBergeron) [inline resources/icons/svg/thumb.svg]: _I think_ you can remove the filter, it doesn't look like it's used (same for the st0 class). I'd check in the lib first to see if they toggle the class somewhere

File: resources/icons/svg/thumb.svg [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  <path d="M1.6,0h24.8C27.3,0,28,0.7,28,1.6v10.8c0,0.9-0.7,1.6-1.6,1.6H1.6C0.7,14,0,13.3,0,12.4V1.6
+					C0,0.7,0.7,0,1.6,0z"/>
+  <path class="st1" d="M1.6,0h24.8C27.3,0,28,0.7,28,1.6v10.8c0,0.9-0.7,1.6-1.6,1.6H1.6C0.7,14,0,13.3,0,12.4V1.6
```
Comment (GermainBergeron) [inline resources/icons/svg/thumb.svg]: instead of using class="st1" I would put the fill here

File: resources/icons/svg/thumb.svg [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+					C0,0.7,0.7,0,1.6,0z"/>
+  <g transform="translate(8.800000, 2.470588)">
+    <path id="Line" class="st2" d="M5.2,0.4v8.2"/>
+    <path id="Line_1_" class="st2" d="M9.2,1.2v6.6"/>
+    <path id="Line_2_" class="st2" d="M1.2,1.2v6.6"/>
```
Comment (GermainBergeron) [inline resources/icons/svg/thumb.svg]: You can remove the Ids (Line, Line_1_ and Line_2_)

File: scss/variables.scss [hunk 7]
```diff
@@ -370,3 +371,22 @@ $limit-progress-bar-height: 10px;
...
+$dot-height: 4px;
+$dot-width: 3px;
+$dot-active-height: 8px;
+$handle-width: 34px;
+$handle-height: 18px;
+$handle-margin-top: -6px;
+$handle-margin-left: -18px;
+$handle-background-size: 34px 18px;
+$handle-box-shadow: 0 0 1px 0 $dark-blue;
+$handle-border-radius: 4px;
```
Comment (GermainBergeron) [inline scss/variables.scss]: Can you prepend each variable with `$slider-` this way we won't confused them with other things

**Pull request merged**